libmscore/style.h

```diff
@@ -1096,6 +1096,99 @@ enum class Sid {
+//     e) _isMetric determines the base page size, A4 or Letter
+//------------------------------------------------------------------------------
+struct PageUnit {
+      const char* _key;   // never translated
+      const char* _name;
+      const char* _suffix;
+      bool _isMetric;     // used to configure default page size
+      int _decimals;      // number of decimal places for QDoubleSpinBox
+      qreal _step;        // for spin widgets in specific units
+      qreal _stepSpatium; // ditto
+      qreal _max;         // max width and height value
+      qreal _factor;      // conversion-to-points factor
+      qreal _inchFactor;  // conversion-to-inches factor for 3.01 and earlier ///4.00 obsolete
+
+public:
+      const char* key()    const { return _key;            }
+      const char* name()   const { return _name;           }
+      const char* suffix() const { return _suffix;         }
+      bool isMetric()      const { return _isMetric;       }
+      int decimals()       const { return _decimals;       }
+      qreal step()         const { return _step;           }
+      qreal stepSpatium()  const { return _stepSpatium;    }
+      qreal max()          const { return _max;            }
+      qreal factor()       const { return _factor;         }
+      qreal paintFactor()  const { return _factor * DPI_F; } // converts spatium to/from user units
+      qreal inchFactor()   const { return _inchFactor;     } ///4.00 obsolete - converts user units to/from inches
+      };
+
+//------------------------------------------------------------------------------
+//   pageUnits - array of PageUnit, contents and order mimic QPageLayout::Unit
+//------------------------------------------------------------------------------
+const PageUnit pageUnits[] = { 
+      { "Millimeters", QT_TRANSLATE_NOOP("unitName", "Millimeters"), QT_TRANSLATE_NOOP("unitSuffix", "mm"), true,  2, 1.00, 0.200,  9999.99, PPI / INCH,   1.0 / INCH },
+      { "Points",      QT_TRANSLATE_NOOP("unitName", "Points"),      QT_TRANSLATE_NOOP("unitSuffix", "pt"), false, 0, 1.00, 0.200, 99999.99, 1.0,          1.0 / PPI  },
+      { "Inches",      QT_TRANSLATE_NOOP("unitName", "Inches"),      QT_TRANSLATE_NOOP("unitSuffix", "in"), false, 2, 0.05, 0.005,  2000.99, PPI,          1.0 },
+      { "Picas",       QT_TRANSLATE_NOOP("unitName", "Picas"),       QT_TRANSLATE_NOOP("unitSuffix", "p" ), false, 2, 1.00, 0.100,  9999.99, PICA,         1.0   / (PPI / PICA) },
+      { "Didot",       QT_TRANSLATE_NOOP("unitName", "Didot"),       QT_TRANSLATE_NOOP("unitSuffix", "dd"), true,  0, 1.00, 0.005, 99999.99, DIDOT,        DIDOT / PPI          },
+      { "Cicero",      QT_TRANSLATE_NOOP("unitName", "Cicero"),      QT_TRANSLATE_NOOP("unitSuffix", "c" ), true,  2, 1.00, 0.100,  9999.99, DIDOT * PICA, DIDOT / (PPI / PICA) },
+      };
+
+// For human readable .mscx files
+const QString pageOrient[] = { QString("Portrait"), QString("Landscape") };
```

Comment:
This is not necessary for this PR.  It will be for a later PR.  I'll remove it in the next push.